contrib/gorilla/mux/mux.go

```diff
@@ -116,7 +116,7 @@ func (r *Router) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		spanopts = append(spanopts, headerTagsFromRequest(req))
 	}
 	resource := r.config.resourceNamer(r, req)
-	httputil.TraceAndServe(r.Router, &httputil.TraceConfig{
+	httptrace.TraceAndServe(r.Router, &httptrace.TraceConfig{
```

Comment:
```suggestion
	httptrace.TraceAndServe(r.Router, &httptrace.ServeConfig{
```
I think this name needs to change within the current context. WDYT?

Reply:
ok, I' will change it